src/components/PeopleTable.tsx

```diff
@@ -1,683 +1,106 @@
-            >
-              Maria van Brussel
-            </a>
-          </td>
-          <td>f</td>
-          <td>1801</td>
-          <td>1834</td>
-          <td>
-            <a
-              className="has-text-danger"
-              href="#/people/joanna-de-causmaecker-1762"
-            >
-              Joanna de Causmaecker
-            </a>
-          </td>
-          <td>
-            <a href="#/people/jan-frans-van-brussel-1761">
-              Jan Frans van Brussel
-            </a>
-          </td>
-        </tr>
-
-        <tr data-cy="person">
-          <td>
-            <a href="#/people/carolus-haverbeke-1832">Carolus Haverbeke</a>
-          </td>
-          <td>m</td>
-          <td>1832</td>
-          <td>1905</td>
-          <td>
-            <a
-              className="has-text-danger"
-              href="#/people/maria-van-brussel-1801"
-            >
-              Maria van Brussel
-            </a>
-          </td>
-          <td>
-            <a href="#/people/carel-haverbeke-1796">Carel Haverbeke</a>
-          </td>
-        </tr>
-
-        <tr data-cy="person">
-          <td>
-            <a
-              className="has-text-danger"
-              href="#/people/maria-sturm-1835"
-            >
-              Maria Sturm
-            </a>
-          </td>
-          <td>f</td>
-          <td>1835</td>
-          <td>1917</td>
-          <td>Seraphina Spelier</td>
-          <td>Charles Sturm</td>
-        </tr>
+type Props = {
+  people: Person[],
+};
 
-        <tr data-cy="person">
-          <td>
-            <a
-              className="has-text-danger"
-              href="#/people/emma-de-milliano-1876"
-            >
-              Emma de Milliano
-            </a>
-          </td>
-          <td>f</td>
-          <td>1876</td>
-          <td>1956</td>
-          <td>Sophia van Damme</td>
-          <td>Petrus de Milliano</td>
-        </tr>
+export const PeopleTable: React.FC<Props> = ({
+  people,
+}) => {
+  let visiblePeople = [...people];
+
+  const { searchParams } = useContext(SearchParamsContext);
+
+  const sort = searchParams.get('sort');
+  const order = searchParams.get('order');
+  const currentSexFilter = searchParams.get('sex');
+  const currentQueryFilter = searchParams.get('query');
+  const currentCenturiesFilter = searchParams.getAll('centuries');
+
+  if (currentSexFilter) {
+    visiblePeople = visiblePeople.filter(({ sex }) => sex === currentSexFilter);
+  }
+
+  if (currentQueryFilter) {
+    visiblePeople = visiblePeople.filter(({ name }) => {
+      const nameLowercased = name.toLowerCase();
+      const currentQueryFilterLowercased = currentQueryFilter.toLowerCase();
+
+      return nameLowercased.includes(currentQueryFilterLowercased);
+    });
+  }
+
+  if (currentCenturiesFilter.length) {
+    visiblePeople = visiblePeople.filter(({ born, died }) => {
+      const bornCentury = Math.ceil(born / 1000);
+      const diedCentury = Math.ceil(died / 100);
```

Comment:
It seems like there's a typo in the `bornCentury` calculation. It should be `Math.ceil(born / 100)` instead of `Math.ceil(born / 1000)` 🧐